src/reduce.js

```diff
@@ -6,6 +6,40 @@ import {abs, exp, log} from './unary.js';
 import {sizeOfShape, Scalar, Tensor} from './lib/tensor.js';
 import {validateReduceParams} from './lib/validate-input.js';
 
+export function selectValuesToReduce(input, axes, inputLocation) {
+  validateReduceParams(input, {axes});
+
+  const outputShape = input.shape.slice();
+  for (let i = 0; i < axes.length; ++i) {
+    outputShape[axes[i]] = 1;
+  }
+
+  // Calculate the "strides" across the reduction dimensions given in axes.
+  axes.sort((a, b) => a - b);
```

Comment:
(I'll never understand how JS's `axes.sort()` doesn't just work correctly by default, without the need for an explicit callback, because who wants to sort numbers as if they were strings?? 🙃) \<resolve me\>

Reply:
Updated. Yes, here we can just invoke `axes.sort()`.

Reply:
Oh, I wasn't asking it to be changed, just remarking on the oddity of the lambda, but I understand why the lambda is needed because calling JS `sort()` is so problematic. Given [2,13], JS sorts them as [13,2] :(.

![image](https://github.com/webmachinelearning/webnn-baseline/assets/1809166/87081677-6a39-4035-b0b6-d4e0e63ee298)

Reply:
Roll back using JS sort() with lambda.